### ISSUE TITLE
Deeplabcut not accessing GPU

### ISSUE THREAD

**Issue by natelam21** [DeepLabCut/DeepLabCut#2666]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I've installed a GPU Driver and CUDA, and using nvcc -V works, but I can't access the GPU to do training/analyzing. When I train, the CPU goes up to 100% and the GPU doesn't act at all. 

I've installed the GeForce Driver and CUDA versions (see below), how do I access my GPU? And does the version of pytorch matter? 

### Operating System

operating system Windows 11

### DeepLabCut version

dlc version 3.0.0rc2

### DeepLabCut mode

single animal

### Device type

GPU: NVIDIA GeForce RTX 3050 OEM

### Anything else?

For reference, here's what I get from nvidia-smi and nvcc -V
![image](https://github.com/user-attachments/assets/7055a6e3-aea2-4499-81b8-72e216d0dba5)
![image](https://github.com/user-attachments/assets/e837ec8a-8a5b-4d7f-b128-e861a0bb2778)

However, when I run the following I get this result
![image](https://github.com/user-attachments/assets/fb10f8f8-7f25-4dc9-ba6b-976a66139aad)

Also my version of pytorch: 
![image](https://github.com/user-attachments/assets/f554f318-f40d-42b5-a84e-2db9d2c2e13e)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Hi @natelam21 - to use your GPU with DeepLabCut, all you need is a version of `PyTorch` installed that can use the GPU. Their [installation instructions](https://pytorch.org) explain how to do this. 

The commands you ran to check if `PyTorch` had access to the GPU are correct (`import torch` and then `torch.cuda.is_available()`). If that returns True, then DeepLabCut will be able to use the GPU.

As mentioned on their website for your configuration, try running (`torchaudio` is not needed):

```bash
pip uninstall pytorch torchvision
conda install pytorch torchvision pytorch-cuda=12.1 -c pytorch -c nvidia
```

![torch_install_windows](https://github.com/user-attachments/assets/bd68bf5b-ccd2-413f-b2af-ec57d8e989ab)

I would recommend uninstalling the CPU-versions that are already installed before re-installing PyTorch. If that doesn't work, try doing this in a fresh conda environment (install PyTorch before you install DeepLabCut in your conda environment, as mentioned in our pull request for DLC 3.0.

Please let me know if this works for you :)

**Comment by ronensgv** [comment]
Hi, I have the same problem. Deeplabcut does not use GPU. I have tried all suggestions with no success. Specifically, I removed deeplabcut, istalled PyTorch, and veryfied it workd with torch.cuda.is_available() (got True). The I reinstalled deeplabcut and still no sucess in getting deeplabcut to us GPU. It runs but on CPU only and results are very slow. Any idea what to do next?

**Comment by n-poulsen** [comment]
@ronensgv how are you checking whether the GPU is being used?

**Comment by ronensgv** [comment]
In the task manager. CPU goes up and and GPU syas idle. 

In meanwhile we follow the solution at issue #2623. First we removed everything, deeplab cut and conda and GPU drivers. Then we made sure all needed for GPU is installed as instrcted in https://deeplabcut.github.io/DeepLabCut/docs/installation.html#gpu-support. We also istalled cuDNN manually. It is supposed to be in the anaconda environment but this was not done in the previous istallation. Then we followed the instructions summaries at the bottom of issuse #2623. Then it worked. Thanks a lot!
